pkg/test/environment.go

```diff
@@ -89,7 +89,7 @@ func NewEnvironment(options ...option.Function[EnvironmentOptions]) *Environment
 	ctx, cancel := context.WithCancel(context.Background())
 
 	os.Setenv(system.NamespaceEnvKey, "default")
-	version := version.MustParseSemantic(strings.Replace(env.WithDefaultString("K8S_VERSION", "1.30.x"), ".x", ".0", -1))
+	version := version.MustParseGeneric(strings.Replace(env.WithDefaultString("K8S_VERSION", "1.30.x"), ".x", ".0", -1))
```

Comment:
intentional? What's the implication of this change

Reply:
This is something I've run into while doing local dev pretty often. I meant to remove for this change and will revisit.